cssgunc/CSPortal
# [Auth] Reformatted the sign up and sign in forms

## Summary 

Restyled the authentication forms (sign in and sign up) according to the styles in Figma. Also added formatting for the error message that appears for invalid authentication details. Also changed our website background to white.

## To test 

1. Check out the auth-forms branch `git checkout auth-forms`
2. Run `npm start` in your command line inside the RTCPortal root directory
3. Sign out to see the sign in form
4. Click "sign up" to see the sign up form

## Test cases with error message examples

**1. Desktop sign in:**

![Screen Shot 2020-10-17 at 6 42 55 PM](https://user-images.githubusercontent.com/43911326/96354867-5c5c4380-10a9-11eb-8cb9-2e9aa23a8cc4.png)
![Screen Shot 2020-10-17 at 6 41 56 PM](https://user-images.githubusercontent.com/43911326/96354856-4a7aa080-10a9-11eb-9ca0-10447f15b28e.png)

**2. Desktop sign up:**

![Screen Shot 2020-10-17 at 6 42 48 PM](https://user-images.githubusercontent.com/43911326/96354863-55cdcc00-10a9-11eb-9bbc-43fad1f92c06.png)
![Screen Shot 2020-10-17 at 6 42 08 PM](https://user-images.githubusercontent.com/43911326/96354858-4d759100-10a9-11eb-860b-8cbc24043199.png)

**3. Mobile:**

![mobile sign in](https://user-images.githubusercontent.com/43911326/96354845-4058a200-10a9-11eb-9f27-d63377762eb0.gif)
![Screen Shot 2020-10-17 at 6 41 46 PM](https://user-images.githubusercontent.com/43911326/96354854-45b5ec80-10a9-11eb-9a6f-3110994111d9.png)
